docs/MixPanel-batchsource.md

```diff
@@ -0,0 +1,22 @@
+# MixPanel batch source
+
+Description
+-----------
+This plugin used to fetch MixPanel events.
+
+Properties
+----------
+### General
+
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**API Secret:** API secret used to authenticate in MixPanel API.
+
+**From date:** Start date for reports data.
+
+**To date:** End date for reports data.
+### Advanced
+
+**Events:** Comma separated list of events you would like to get data on.
+
+**Filter:** Expression to filter events by.
```

Comment:
what can be the value of a filter?

Reply:
Syntax is complex, added link to official documentation.